app/assets/javascripts/components/story/ExpandedStory/index.js

```diff
@@ -68,83 +48,21 @@ export class ExpandedStory extends React.Component {
           canSave={Story.canSave(story)}
           canDelete={Story.canDelete(story)}
         />
-        {
-          !Story.isNew(story) ?
-            <ExpandedStoryHistoryLocation story={story} />
-            : null
-        }
-        <ExpandedStoryTitle
-          story={story}
-          titleRef={this.titleRef}
-          onEdit={(newTitle) => editStory(story.id, { title: newTitle })}
-        />
-
-        <div className="Story__flex">
-          <ExpandedStoryEstimate story={story}
-            onEdit={(newAttributes) => editStory(story.id, newAttributes)}
-          />
-
-          <ExpandedStoryType story={story}
-            onEdit={(newAttributes) => editStory(story.id, newAttributes)}
-          />
-        </div>
-
-        <ExpandedStoryRequestedBy
-          story={story}
-          users={users}
-          onEdit={(userId) => editStory(story.id, { requestedById: userId })}
-        />
-
-        <ExpandedStoryOwnedBy
-          story={story}
-          users={users}
-          onEdit={(userId) => editStory(story.id, { ownedById: userId })}
-        />
-
-        <ExpandedStoryState
-          story={story}
-          onEdit={(newAttributes) => editStory(story.id, newAttributes)}
-        />
-
-        <ExpandedStoryLabels
-          onAddLabel={(label) => addLabel(story.id, label)}
-          story={story}
-          projectLabels={project.labels}
-          onRemoveLabel={(labelName) => removeLabel(story.id, labelName)}
-          onEdit={(value) => editStory(story.id, { labels: value })}
-        />
-
-        <ExpandedStoryDescription
-          story={story}
-          onEdit={(newAttributes) => editStory(story.id, newAttributes)}
-        />
 
         {
-          !Story.isNew(story) ?
-            <Fragment>
-              <ExpandedStoryAttachments
-                story={story}
-                onFailure={(error) => storyFailure(story.id, error)}
-                startLoading={() => setLoadingStory(story.id)}
-                onAdd={(attachment) => addAttachment(story.id, project.id, attachment)}
-                onDelete={(documentId) => removeAttachment(story.id, documentId)}
-              />
-
-              <ExpandedStoryNotes
-                story={story}
-                projectId={project.id}
-                onDelete={(noteId) => deleteNote(project.id, story.id, noteId)}
-                onCreate={(note) => createNote(project.id, story.id, { note })}
-              />
-
-              <ExpandedStoryTask
-                story={story}
-                onToggle={(task, status) => toggleTask(project.id, story, task, status)}
-                onDelete={(taskId) => deleteTask(project.id, story.id, taskId)}
-                onSave={(task) => createTask(project.id, story.id, task)}
-              />
-            </Fragment>
-            : null
+          Story.isRelease(story._editing.storyType)
```

Comment:
Doesn't the `Story.isRelease` method accept a story instance as parameter rather than the `storyType`?